var/spack/repos/builtin/packages/cp2k/package.py

```diff
@@ -996,6 +1002,9 @@ def cmake_args(self):
             self.define_from_variant("CP2K_USE_SPLA", "spla"),
             self.define_from_variant("CP2K_USE_QUIP", "quip"),
             self.define_from_variant("CP2K_USE_MPI_F08", "mpi_f08"),
+            self.define_from_variant("CP2K_USE_TREXIO", "trexio"),
+            self.define_from_variant("CP2K_USE_GRPP", "grpp"),
+            self.define_from_variant("CP2K_USE_GRPP", "trexio"),
```

Comment:
I think this needs better handling, otherwise `+grpp ~trexio` will result in conflicting values for `CP2K_USE_GRPP`. Or am I missing something (like order dependence in CMake arguments or similar)?

Reply:
I modified the logic. GRPP is 
- ON when `+grpp`, `+trexio+grpp`
- OFF when `~grpp` independently of the status of `trexio`